.vscode/settings.json

```diff
@@ -23,7 +23,8 @@
   "typescript.preferences.quoteStyle": "single",
   "typescript.enablePromptUseWorkspaceTsdk": true,
   "typescript.preferences.preferTypeOnlyAutoImports": true,
-  "jest.jestCommandLine": "yarn jest",
+  "jest.jestCommandLine": "yarn jest --testTimeout=1000000",
```

Comment:
Do we want to keep that ?

Reply:
I think we do. Without it we cannot run the IT in VSCode :S 
But the value of the timeout can be reviewed of course

Reply:
16m40s is maybe a lot

Reply:
Give me a number I will put it :D